sdk/identity/test-resources-post.ps1

```diff
@@ -0,0 +1,113 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# IMPORTANT: Do not invoke this file directly. Please instead run eng/New-TestResources.ps1 from the repository root.
+
+param (
+  [hashtable] $DeploymentOutputs
+)
+
+# # If not Linux, skip this script.
+# if ($isLinux -ne "Linux") {
+#   Write-Host "Skipping post-deployment because not running on Linux."
+#   return
+# }
+
+$ErrorActionPreference = 'Stop'
+$PSNativeCommandUseErrorActionPreference = $true
+
+$webappRoot = "$PSScriptRoot/identity/test/integration" | Resolve-Path
+$workingFolder = $webappRoot;
+
+Write-Host "Working directory: $workingFolder"
+
+az login --service-principal -u $DeploymentOutputs['IDENTITY_CLIENT_ID'] -p $DeploymentOutputs['IDENTITY_CLIENT_SECRET'] --tenant $DeploymentOutputs['IDENTITY_TENANT_ID']
+az account set --subscription $DeploymentOutputs['IDENTITY_SUBSCRIPTION_ID']
+
+Write-Host "Sleeping for a bit to ensure container registry is ready."
+Start-Sleep -s 60
```

Comment:
60 seconds seems a little expensive, how did we arrive at 60?

Reply:
I took that from .NET  and Python test setup. Let me check with them. Thanks for pointing out.